roles/deploy/handlers/main.yml

```diff
@@ -50,3 +50,8 @@
     state: restarted
   become: yes
   become_user: root
+
+- name: update whenever
+  command: bash -lc "bundle exec whenever --update-crontab RAILS_ENV={{ rails_env }}"
```

Comment:
We should have the RAILS_ENV already available on the machine but this feels safer, so agreed. What do others think?